### PR TITLE
chore(pg): Refactor writing a bit

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Im in the middle of a refactor of destinations. so destinations mostly on freeze right now as otherwise I wont be able to solve conflicts.

File: plugins/destination/postgresql/client/write.go [hunk 1]
```diff
@@ -31,52 +32,37 @@ func (c *Client) Write(ctx context.Context, table string, data map[string]interf
 	return nil
 }
 
-func insert(table string, data map[string]interface{}) (string, []interface{}) {
-	var sb strings.Builder
+func insert(table string, data map[string]interface{}) (string, []any) {
```
Comment (bbernays) [inline plugins/destination/postgresql/client/write.go:35]: I wouldn't switch `[]interface{}` to `[]any` unless we are going to make the change everywhere

**Review by bbernays — COMMENTED** [review]
Overall looks good one nit

File: plugins/destination/postgresql/client/write.go (hunk 1 above)
Comment (candiduslynx) [reply plugins/destination/postgresql/client/write.go:35]: for now, `any` is just an alias of `interface{}`, so in this case I propose to gradually convert to `any` (as it's a bit cleaner)

**Comment by yevgenypats** [comment]
Closing as this will be changed completely by https://github.com/cloudquery/plugin-sdk/pull/298 and https://github.com/cloudquery/cloudquery/pull/3176

**Pull request closed without merging**